templates/aem-tools/content-healthcheck.py.epp

```diff
@@ -1,31 +1,50 @@
 #!/usr/bin/env python
-import urllib, json, requests, urllib3, boto3, sys, argparse, logging, socket, re, random
+import urllib, json, requests, urllib3, boto3, sys, argparse, logging, socket, re, random, time
 from botocore.exceptions import ClientError
 from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from HTMLParser import HTMLParser
+from urlparse import urlparse
 
 requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
 
+DISPATCHER_HEALTH_METRIC = 'contentHealthCheck'
+REQUEST_LATENCY_METRIC ='contentRequestLatency'
+TIMEOUT_METRIC ='contentRequestTimeOut'
+CONNECTION_ERROR_METRIC ='contentRequestConnectionError'
+
+UNHEALTHY_MESSAGE = 'UNHEALTHY CloudWatch Metric Sent'
+HEALTHY_MESSAGE = 'HEALTHY CloudWatch Metric Sent'
+SUCCESSFUL_VALUE = 1;
+UNSUCCESSFUL_VALUE = 0;
+
+LOG_LEVEL = logging.INFO
+
+metric_data = []
+instance_variables = {}
+
 try:
     import coloredlogs
     coloredlogs.install(
         isatty = True,
         show_name = False,
         show_severity = False,
-        level = logging.NOTSET,
+        level = LOG_LEVEL,
         severity_to_style = { 'DEBUG': {'color': 'blue'}},
     )
 except:
     logging.basicConfig(
         stream = sys.stdout,
         format = '%(asctime)s ' + socket.gethostname() + ' %(levelname)-8s %(message)s',
         datefmt = "%Y-%m-%d %H:%M:%S",
-        level = logging.NOTSET,
+        level = LOG_LEVEL,
     )
 log = logging.getLogger(__name__)
 
 local_path = '<%= $tmp_dir %>/content-healthcheck-descriptor.json'
-session = boto3.session.Session()
-region = session.region_name
+
+#set region as no default region
+region='ap-southeast-2'
```

Comment:
This region value needs to be injected into the content health check EPP template, and in turn, the param should be injected via `publish_dispatcher::aws_region` with the value retrieved from `common::aws_region` and then set in hieradata file.